test/cmake_install_test/main.cpp

```diff
@@ -0,0 +1,13 @@
+// Copyright 2018 Peter Dimov
+// Copyright 2020 Richard Hodges
+// Distributed under the Boost Software License, Version 1.0.
+// See accompanying file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt
+
+#include <boost/json.hpp>
+#include <iostream>
+
+int main()
+{
+    boost::json::error_code err = boost::json::error::expected_colon;
+    std::cout << err.message() << std::endl;
```

Comment:
This doesn't exercise any functionality from the compiled library, so may hide link errors.

Reply:
isn't the compiled version just an artefact of trying to reduce travis build times? Will anyone actually use it?

Reply:
No, the point of the compiled version is to reduce everyone's build time, by removing as much material as possible from the headers.